opm/grid/LookUpData.hh

```diff
@@ -431,6 +464,49 @@ Opm::LookUpData<Grid,GridView>::getFieldPropIdx(const int& elemIdx) const
     }
 }
 
+template<typename Grid, typename GridView>
+template<typename GridType>
+typename std::enable_if_t<!std::is_same_v<GridType,Dune::CpGrid>,int>
+Opm::LookUpData<Grid,GridView>::getParentFaceIdxFromLgrBoundaryFace(const int& faceIdx) const
+{
+    static_assert(std::is_same_v<Grid,GridType>);
+    // Check there are no LGRs. LGRs (level>0) only supported for CpGrid.
+    assert(gridView_.grid().maxLevel() == 0);
+    return faceIdx;
```

Comment:
I fear there is no working implementation for non-CpGrids here. At least no easy or cheap one. That is  a pity.

I think just returning an index is really dangerous. At least we should throw an exception with a meaningful error message here.

Reply:
I see, the idea behind was to have such a method, "getParentFAce...", defined for all type of grids. Then, for non CpGrid grids, would be better to throw instead of actually returning the same face index (or face)